llm/codellama/README.md

```diff
@@ -0,0 +1,221 @@
+# Serving Code Llama 70B with vLLM and SkyServe
```

Comment:
How about leading with SkyPilot / SkyPilot Serving?  In case this creates confusion.  In the main text we can clarify SkyServe = SkyPilot Serving.